source/connect.txt

```diff
@@ -9,5 +9,147 @@ Connect to a Deployment
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+
+Ensure that a MongoDB deployment is running before attempting to 
```

Comment:
[suggest] This is easy to scroll through / miss. Possibly make this an `important` or add a `Prerequisite` heading?